### DISCUSSION TITLE
Hello, is the socks5 proxy format I use also the same protocol://ip:port?

### DISCUSSION THREAD

**Discussion by qwr6868** [MRColorR/money4band#81] [General]
Hello, is the socks5 proxy format I use also the same protocol://ip:port?

**Comment by MRColorR** [comment]
hello, and thank you for reaching out 🙂
yes, you can use protocol://username:password@ip:port or protocol://ip:port depending on your case

**Reply by qwr6868** [reply]
Hello, I used socks5 to run EARNFM and REPOCKET, but it didn’t work.

**Reply by qwr6868** [reply]
They are running but socks5 running them has no use

**Reply by MRColorR** [reply]
Could you enter in some of the running m4b docker containers (using `docker exec -it <podname> bash`) and run the command to check their associated public IP (use `curl icanhazip.com` or `wget -q -O - icanhazip.com 2>&1`?  

You should be able to check if they use the proxy or not this way.

**Reply by qwr6868** [reply]
OK, solved, thank you

**Comment by qwr6868** [comment]
ok thank
